mockall/tests/mock_struct.rs

```diff
@@ -29,52 +29,45 @@ mod checkpoint {
     #[test]
     fn expect_again() {
         let mut mock = MockFoo::new();
-        mock.expect_foo()
-            .returning(|_| 5)
-            .times(1..3);
+        mock.expect_foo().returning(|_| 5).times(1..3);
```

Comment:
Could you please revert the unrelated formatting changes to this file?

Reply:
I've re-done commit 65bfd7d as 9af1d7d which should not include format change. Sorry it slipped in...